### ISSUE TITLE
"Do not cache" tag

### ISSUE THREAD

**Issue by wbrowar** [craftapi/htmlcache#10]
Hey guys,

separate from a possible hook for cache busting (or maybe as an option for it), could you create a tag that we can drop onto page templates that tells HTML Cache not to cache any page that the tag is found on?

This way, if HTML Cache is processing the page, but is sees this twig tag anywhere, it could just stop what it's doing and let the page process normally.

**Comment by webremote** [comment]
Yeah, this has been thorn in my eye for a while now, upgrading it to
critical bug. Will fix it asap. Would it be flexible if you could add
parameters to the twig that could trigger an event? That way, you should be
able to hook on quite a lot.. busting, exceptions, environment related
stuff.. would be a design thing: per locale cache or not (is an important
feature imho), any triggers that should bust a specific tag, time related
(differs per area), and a bunch of other variables that you can hook on.

**Comment by MattiJarvinen** [comment]
I guess you could check for {% header "Pragma: no-cache" %} in service´s createCacheFile?

**Comment by davidyang** [comment]
I just submitted pull request #46 to add this feature.

**Comment by Ottergoose** [comment]
I did a bit of tinkering; quick solution I arrived at was to set a custom header and update the logic in pull request #46 to look for it; "no-cache" was getting sent on all pages by default and prevented caching everywhere.

{% header "X-CraftAPI-HTMLCache: no-craftapi-htmlcache" %}
